superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/plugin/buildQuery.ts

```diff
@@ -32,7 +36,12 @@ import { buildQueryContext, QueryFormData } from '@superset-ui/core';
  * it is possible to define post processing operations in the QueryObject, or multiple queries
  * if a viz needs multiple different result sets.
  */
-export default function buildQuery(formData: QueryFormData) {
+export default function buildQuery(formData: CccsGridQueryFormData) {
+  formData = {
+    ...formData,
+    ...DEFAULT_FORM_DATA,
+  };
+  console.log('formData via buildQuery.ts', formData);
```

Comment:
Won't block the merge but do you want to keep this console.log?

Reply:
Removed that console.log and the one that was already present in transformProps!